README.md

```diff
@@ -1,486 +1,511 @@
-# Dynatrace OneAgent SDK for Java

```

Comment:
It seems there is some kind of line ending issue here.

Reply:
Hm, weird. Does it matter though?